sds_data_manager/stacks/indexer_lambda_stack.py

```diff
@@ -128,7 +128,7 @@ def __init__(
                     "bucket": {"name": [data_bucket.bucket_name]},
                     "object": {
                         "key": [
-                            {"prefix": f"imap/{instrument}/l0/"}
+                            {"prefix": f"imap/{instrument}/"}
```

Comment:
The rule names above this should be updated since it isn't just l0 anymore. Also, it doesn't appear that `instrument` is used for anything inside here. Would it make sense to just have a single prefix that is listening on `imap/`

Reply:
we could make it `imap/` but I was wondering if we will have calibrations or spice inside `imap/` too.

Reply:
Good point! Calibrations might also be within the instrument level though, so would be caught by this. I wonder if that means we need a lot more prefixes to distinguish between all of that, or if we should do a general prefix `imap/` and then do the sorting/filtering within the lambda that gets triggered...

Fine to leave it as-is for now.

Reply:
true. we have already captured `TODO` in the indexer lambda's `s3_event_handler()`. I could change this rule to be `imap/` and then we can handle that in the indexer lambda. That sounds good to me.